independent-publisher-connectors/Rijksmuseum/apiDefinition.swagger.json

```diff
@@ -0,0 +1,836 @@
+                            "description": "height",
+                            "title": "Height of Web Image",
+                            "x-ms-visibility": "advanced"
+                          },
+                          "url": {
+                            "type": "string",
+                            "description": "url",
+                            "title": "Web Image URL"
+                          }
+                        },
+                        "description": "headerImage"
+                      },
+                      "productionPlaces": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        },
+                        "description": "productionPlaces"
+                      }
+                    }
+                  },
+                  "description": "artObjects"
+                },
+                "facets": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "facets": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "key": {
+                              "type": "string",
+                              "description": "key"
+                            },
+                            "value": {
+                              "type": "integer",
+                              "format": "int32",
+                              "description": "value"
+                            }
+                          }
+                        },
+                        "description": "facets"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name",
+                        "title": "Facet Name",
+                        "x-ms-visibility": "advanced"
+                      },
+                      "otherTerms": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "otherTerms",
+                        "title": "Facet Other Terms",
+                        "x-ms-visibility": "advanced"
+                      },
+                      "prettyName": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "prettyName",
+                        "title": "Facet Pretty Name",
+                        "x-ms-visibility": "advanced"
+                      }
+                    }
+                  },
+                  "description": "facets"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search the Collection",
+        "operationId": "getCollection",
```

Comment:
@ashes2076 all operationIds should be in Camel Case. Please change to "GetCollection".
Please fix all operationIds in the swagger file.

Reply:
done